python/lsst/ap/association/diaForcedSource.py

```diff
@@ -277,7 +277,8 @@ def _calibrate_and_merge(self,
         midPointTaiMJD = visit_info.getDate().get(system=DateTime.MJD)
         output_catalog["ccdVisitId"] = ccdVisitId
         output_catalog["midPointTai"] = midPointTaiMJD
-        output_catalog["filterName"] = diff_exp.getFilter().getCanonicalName()
+        # TODO DM-21333: Remove [0] (first character only) workaround
+        output_catalog["filterName"] = diff_exp.getFilter().getCanonicalName()[0]
```

Comment:
DM-21333 is reported as 'Done', so you may want to replace the TODO with when RFC-730 is "Implemented"

Reply:
Could you add a comment about what quantity you are trying to retrieve here (i.e. band?, physical filter?), since that will help the conversion effort to the new filter.

Reply:
Good point, I will reference DM-27170 instead, which appears to be one of the final pieces of RFC-730. This issue appears in a few other places in ap_association as well, so I will update those accordingly. The goal here is to retrieve a single-character representation of the band.